docs/client/full-api/api/methods.md

```diff
@@ -178,7 +178,10 @@ even if the method's writes are not available yet, you can specify an
 
 `Meteor.apply` is just like `Meteor.call`, except that the method arguments are
 passed as an array rather than directly as arguments, and you can specify
-options about how the client executes the method.
+options about how the client executes the method.  Options permitted are:
```

Comment:
We don't need to list these options here, right? They should be in the API box above.